docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -199,31 +533,46 @@ It is not mandatory to build applications on top of the above samples, however,
 
 CKEditor&nbsp;5 supports {@link getting-started/setup/ui-language multiple UI languages}, and so does the official React component. Follow the instructions below to translate CKEditor&nbsp;5 in your React application.
 
-Similarly to CSS style sheets, both packages have separate translations. Import them as shown in the example below. Then, pass them to the `translations` array inside the `config` prop in the CKEditor 5 component.
+Pass the language you need into the `translations` array inside the configuration in the `useCKEditorCloud`.
```

Comment:
This is not reflected on the snippet below, should it be? :thinking: 

There is only `languages` key.

Reply:
Good catch. I'm guessing it should be the `languages` key, right? @Mati365

Reply:
Maybe it makes sense to rename it to fit editor config.

Reply:
It's named language in editor config, https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editorconfig-EditorConfig.html#member-language . @gorzelinski yep, it should be languages

Reply:
Setting the [language of the ui and/or content](https://ckeditor.com/docs/ckeditor5/latest/getting-started/setup/ui-language.html#setting-the-language-of-the-content) is a bit different thing than [loading the translations](https://ckeditor.com/docs/ckeditor5/latest/getting-started/setup/ui-language.html#loading-additional-languages-from-npm-or-cdn).

Reply:
Renamed to translations